rules/falco_rules.yaml

```diff
@@ -2104,6 +2104,15 @@
 - macro: user_known_non_sudo_setuid_conditions
   condition: user.name=root
 
+# Various probes may be necessary to manage and orchestrate containers and
+# these probes should be excepted out of certain rules.
+# See https://github.com/falcosecurity/falco/issues/310
+- macro: is_container_probe
+  condition: >
+    proc.is_container_healthcheck or
```

Comment:
Can you surround the condition with parentheses? It makes sure you don't end up with ambiguity when using the macro in larger logical expressions.

Reply:
Yup, added it in the latest version. Would've thought they'd be auto-added around macros o.o